### PR TITLE
add: `polkit-git`

### PR DESCRIPTION
**Pull request by WRM-42**

### CODE REVIEW

File: packages/polkit-git/polkit-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name="polkit-git"
+pkgname="polkit"
+gives="polkit"
+repology=("project: ${pkgname}")
```
Comment (wizard-28) [inline packages/polkit-git/polkit-git.pacscript:4]:
```suggestion

```

File: packages/polkit-git/polkit-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  ./autogen.sh
+}
+
+build() {
+  ./configure
+  make -j"$(nproc)"
+}
+
+install() {
+  sudo make install DESTDIR="/usr/src/pacstall/${pkgname}"
```
Comment (wizard-28) [inline packages/polkit-git/polkit-git.pacscript:25]:
```suggestion
  sudo make install DESTDIR="${STOWDIR}/${pkgname}"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by wizard-28** [comment]
The CI fails, fix the pacscript.

**Comment by WRM-42** [comment]
More complicated than I thought. Polkit depends on Mozjc, which is not available on Debian/Ubuntu repos. I'm not familiar with the build system of Mozilla, so I will close this pull request for now

**Pull request closed without merging**